lib/monmo-repl.js

```diff
@@ -217,7 +217,8 @@ function opQuery(){
     .sort({$natural:1})
     .addOption(DBQuery.Option.tailable)
     .addOption(DBQuery.Option.noTimeout)
-    .addOption(DBQuery.Option.awaitdata);
+    .addOption(DBQuery.Option.awaitdata)
+    .addOption(DBQuery.Option.oplogReplay);
```

Comment:
This supersecret query boosts querying speed against oplog.rs.ts significantly!!!!!!!!